index.html

```diff
@@ -83,95 +113,181 @@
+  var width = 800 - margin.right - margin.left;
+  var height = 600 - margin.top - margin.bottom;
 
   // FETCH DATA
-
   d3.json('data.json', function (error, json) {
     if (error) { throw error; }
 
     data = json;
 
-    setup();
-    update();
+    var chart = new Chart(); 
+
+  // EVENT HANDLERS
+  d3.select('#year-range').on('change', function () {
+    options.yearRange = d3.event.target.value;
+    chart.update();
+    });
   });
 
-  function setup() {
-    // SVG
+  function Chart() { 
+    var chart = this; 
 
-    svg = d3.select('#chart')
+    // SVG
+    chart.svg = d3.select('#chart')
       .append('svg')
       .attr('width', width + margin.right + margin.left)
       .attr('height', height + margin.top + margin.bottom)
       .append('g')
       .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
-
+    
     // SCALES
-
-    x = d3.scale.linear()
-      .domain([0, d3.max(data, function (d) { return d.unemployment; })])
+    chart.x = d3.scale.linear()
+      .domain(d3.extent(data, function (a) {return a.costs; }))
       .range([0, width])
       .nice();
 
-    y = d3.scale.linear()
-      .domain(d3.extent(data, function (d) { return d.inflation; }))
+    chart.y = d3.scale.linear()
+      .domain(d3.extent(data, function (a) {return a.visits; }))
       .range([height, 0])
       .nice();
-
+    
     // AXES
-
-    // EVENT HANDLERS
-
-    d3.select('#year-range').on('change', function () {
-      options.yearRange = d3.event.target.value;
-      update();
-    });
+    var formatNumber = d3.format(".0f");
+
+    var xAxis = d3.svg.axis()
+      .scale(chart.x)
+      .tickFormat(formatCurrency)
+      .tickSize(height)
+      .orient("bottom"); 
+    
+    svg.append("g")
+      .append("text")
+      .attr("x", 190)
+      .attr("y", 580)
+      .style("text-anchor", "start")
+      .style("font-family", "Open Sans Condensed, Helvetica, sans-serif")
+      .style("font-weight", "bold")
+      .text("Standardized, Risk-Adjusted Per Capita Medicare Spending");
+    
+    var gx = svg.append("g")
+      .attr("class", "x axis")
+      .call(xAxis);
+    
+    gx.selectAll("g").filter(function(a) {return a;})
+      .classed("minor", true);
+    
+    var yAxis = d3.svg.axis()
+      .scale(chart.y)
+      .tickSize(width)
+      .orient("right");
+    
+    svg.append("g")
+      .append("text")
+      .attr("transform","rotate(-90)")
+      .attr("x", -250)
+      .attr("y", -60)
+      .attr("dy", ".71em")
+      .style("text-anchor", "middle")
+      .style("font-family", "Open Sans Condensed, Helvetica, sans-serif")
+      .style("font-weight", "bold")
+      .text("ER Visits per 100K");
+    
+    var gy = svg.append("g")
+      .attr("class", "y axis")
+      .call(yAxis);
+    
+    gy.selectAll("g").filter(function(a) {return a;})
+      .classed("minor", true);
+    
+    gy.selectAll("text")
+      .attr("x", -30);
+
+    chart.tooltip = d3.tip()
```

Comment:
Note that you haven't included the d3-tip library. Add this line below the `<script>` tag that loads d3, but above the one that contains your custom Javascript:

`<script type="text/javascript" src="https://npmcdn.com/d3-tip@0.6.7/index.js"></script>`